charts/nutanix-csi-storage/templates/ntnx-csi-controller-deployment.yaml

```diff
@@ -74,10 +92,10 @@ spec:
           image: {{ .Values.sidecars.resizer.image }}
           imagePullPolicy: {{ .Values.imagePullPolicy }}
           args:
-            - --v=5
+            - --v=2
```

Comment:
Remove the limits on all containers, just keep the requests as in master branch.